domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -436,9 +455,9 @@ public static void closeAllClients() {
     PG_POOLS.clear();
   }
 
-  static PgConnectOptions createPgConnectOptions(JsonObject sqlConfig) {
+  static PgConnectOptions createPgConnectOptions(JsonObject sqlConfig, String desiredHost) {
     PgConnectOptions pgConnectOptions = new PgConnectOptions();
-    String host = sqlConfig.getString(HOST);
+    String host = sqlConfig.getString(desiredHost);
```

Comment:
Can we be sure that the hostname is different and the port is the same?
Or is a different port on the same host a valid use case?

    pgproxy:8000
    pgproxy:8001

Reply:
As far as I know, if using RDS then the "reader" port will be the same. But we could code for the case of the port being different. Is this a likely scenario?